darshan-util/pydarshan/darshan/experimental/plots/plot_dxt_heatmap.py

```diff
@@ -0,0 +1,319 @@
+import os
+import sys
+import glob
+import argparse
+
+import darshan
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
```

Comment:
when this script is executed from repo path `darshan/darshan-util/pydarshan` via for example `python darshan/experimental/plots/plot_dxt_heatmap.py /home/tyler/LANL/rough_work/darshan/treddy_mpi-io-test_id545458_6-3-43062-1247823192538596076_13.darshan`

it will error out because of #384 

```
Traceback (most recent call last):
  File "darshan/experimental/plots/plot_dxt_heatmap.py", line 8, in <module>
    import seaborn as sns
  File "/home/tyler/python_37_venv/lib/python3.7/site-packages/seaborn/__init__.py", line 2, in <module>
    from .rcmod import *  # noqa: F401,F403
  File "/home/tyler/python_37_venv/lib/python3.7/site-packages/seaborn/rcmod.py", line 5, in <module>
    import matplotlib as mpl
  File "/home/tyler/github_projects/darshan/darshan-util/pydarshan/darshan/experimental/plots/matplotlib.py", line 8, in <module>
    import matplotlib.pyplot as plt
ModuleNotFoundError: No module named 'matplotlib.pyplot'; 'matplotlib' is not a package
```

Either someone should open a separate PR to remove that module or it should be temporarily removed here as it is in Jakob's PR so that the code can be invoked that way.

Reply:
So I would suggest to merge #388. I have since renamed the plot_dxt_heatmap.py to plot_dxt_heatmap2.py so there shouldn't be any conflicts. Once this is ready we can drop the other dxt heatmap. I am also fine with removing it from the other pull request completely.

Reply:
I'll open a PR to fix #384 so we can merge those changes and rebase.

Reply:
https://github.com/darshan-hpc/darshan/pull/388 already made the changes that fixed any matplotlib import problems, and takes care of the fallout across examples. That's why I was suggesting, to go ahead to merge 388 that after making sure it does not conflict with any of the files you touched here ;)